errbot/err.py

```diff
@@ -165,6 +174,16 @@ def main():
     mode_selection.add_argument('-l', '--list', action='store_true', help='list all available backends')
     mode_selection.add_argument('--new-plugin', nargs='?', default=None, const='current_dir',
                                 help='create a new plugin in the specified directory')
+    # storage manipulation
+    mode_selection.add_argument('--storage-set', nargs=1, help='DANGER: Delete the given storage namespace '
+                                                               'and set the python dictionary expression '
+                                                               'passed on stdin.')
+    mode_selection.add_argument('--storage-merge', nargs=1, help='DANGER: Merge in the python dictionary expression '
+                                                                 'passed on stdin into the given storage namespace.')
+    mode_selection.add_argument('--storage-get', nargs=1, help='Dumps the given storage namespace in a '
```

Comment:
nit: you use 3rd person here, but not earlier: delete vs Deletes, Dump vs Dumps.